hyperspy/_signals/eds.py

```diff
@@ -177,25 +178,85 @@ def sum(self, axis=None, out=None):
+            for each dimension specify the new:old pixel ratio
+            e.g. a ratio of 1 is no binning
+                 a ratio of 2 means that each pixel in the new spectrum is
+                 twice the size of the pixels in the old spectrum.
+        crop_str : {'False'}, optional
+            when binning by a non-integer number of pixels it is likely that
+             the final row in each dimension contains less than the full quota to
+             fill one pixel.
+             e.g. 5*5 array binned by 2.1 will produce two rows containing 2.1
+             pixels and one row containing only 0.8 pixels worth. Selection of
+             crop_str = 'True' or crop = 'False' determines whether or not this
+             'black' line is cropped from the final binned array or not.
+
+            *Please note that if crop=False is used, the final row in each
+            dimension may appear black, if a fractional number of pixels are left
+            over. It can be removed but has been left to preserve total counts
+            before and after binning.*
+
+        Returns
+        -------
+        s : Signal subclass
+
+        Examples
+        --------
+        >>> spectrum = hs.signals.EDSTEMSpectrum(np.ones([4, 4, 10]))
+        >>> spectrum.data[1, 2, 9] = 5
+        >>> print(spectrum)
+        <EDXTEMSpectrum, title: dimensions: (4, 4|10)>
+        >>> print ('Sum = ', sum(sum(sum(spectrum.data))))
+        Sum = 164.0
+        >>> scale = [2, 2, 5]
+        >>> test = spectrum.rebin(scale)
+        >>> print(test)
+        <EDSTEMSpectrum, title: dimensions (2, 2|2)>
+        >>> print('Sum = ', sum(sum(sum(test.data))))
+        Sum =  164.0
+
+        """
+
+        spectrum = self.data
+        m = super().rebin(scale, out=out, crop=crop)
+        m = out or m
+
+        if "Acquisition_instrument.SEM.Detector.EDS.real_time" in m.metadata:
+            for i, t in enumerate(m.axes_manager.navigation_axes):
+                m.metadata.Acquisition_instrument.SEM.Detector.EDS.real_time\
+                    *= scale[i]
+        if "Acquisition_instrument.SEM.Detector.EDS.live_time" in m.metadata:
+            for i, t in enumerate(m.axes_manager.navigation_axes):
+                m.metadata.Acquisition_instrument.SEM.Detector.EDS.live_time\
+                    *= scale[i]
+        if "Acquisition_instrument.TEM.Detector.EDS.real_time" in m.metadata:
+            for i, t in enumerate(m.axes_manager.navigation_axes):
+                m.metadata.Acquisition_instrument.TEM.Detector.EDS.real_time\
+                    *= scale[i]
+        if "Acquisition_instrument.TEM.Detector.EDS.live_time" in m.metadata:
+            for i, t in enumerate(m.axes_manager.navigation_axes):
+                m.metadata.Acquisition_instrument.TEM.Detector.EDS.live_time\
+                    *= scale[i]
+        if m.metadata.has_item('Signal,Noise_properties.variance'):
+            if isinstance(m.metadata.Signal.Noise_properties.variance,
```

Comment:
this check is also in the base signal but EDS has the additions of updating all the dwell time functions.